### PR TITLE
01 lee adding removing products from cart - actual pull request!

### PR DESCRIPTION
**Pull request by theomoondev**
Notes: 

1) I had to move the order_items#create action into the products controller to get the “Add to Cart” button to work in `products/show.html.erb`. I renamed the controller action to add_to_cart and added a route to products. It's working now! We can probably refactor the products#add_to_cart action later by moving some of the logic into the models.

2) There is a test failing in `orders_controller_test.rb`, but I will work on this in a separate branch. Please ignore all files related to orders controller for now!

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -1,2 +1,42 @@
 class OrdersController < ApplicationController
```
Comment (theomoondev) [inline app/controllers/orders_controller.rb:1]: Please ignore this file for now! I decided to work on this in a separate branch.

File: test/controllers/orders_controller_test.rb [hunk 2]
```diff
@@ -1,7 +1,45 @@
 require "test_helper"
```
Comment (theomoondev) [inline test/controllers/orders_controller_test.rb:1]: Please ignore this file for now! I decided to work on this in a separate branch.

File: test/test_helper.rb [hunk 3]
```diff
@@ -18,4 +18,8 @@ class ActiveSupport::TestCase
   fixtures :all
```
Comment (theomoondev) [inline test/test_helper.rb:18]: Please ignore this file for now! I decided to work on this in a separate branch.

File: app/controllers/order_items_controller.rb [hunk 4]
```diff
@@ -1,2 +1,41 @@
 class OrderItemsController < ApplicationController
+  before_action only: [:update, :destroy] do
+    find_order_item(params[:id])
+  end
+
+  def update
```
Comment (peachmakkoli) [inline app/controllers/order_items_controller.rb:6]: andadjwdnw

**Review by Catherina87 — APPROVED** [review]
Great job, Lee! :)

**Pull request merged**